doc/architecture/database_schemata.md

```diff
@@ -0,0 +1,115 @@
+
+# Database Schemata
+
+Tag and Quality in this case are enum. 
+Postgres supports [enums](https://www.postgresql.org/docs/current/datatype-enum.html) so it is easy to define a static set of values. 
+
+```sql
+CREATE TYPE tag AS ENUM (
+  'Leaf crops',
```

Comment:
Hm, good point. This could be done in many ways, for example in the service layer translating to German after querying the db or directly in the db. I'll comment it for now.